std/range/package.d

```diff
@@ -213,7 +213,7 @@ if (isBidirectionalRange!(Unqual!Range))
     }
     else
     {
-        static struct Result()
+        static struct Result
```

Comment:
No idea, I've been using 0-arg templates for "please inline me" with great success in std.regex